actionpack/lib/abstract_controller/base.rb

```diff
@@ -14,8 +14,16 @@ class ActionNotFound < StandardError
   # expected to provide their own +render+ method, since rendering means
   # different things depending on the context.
   class Base
+    ##
+    # Returns the body of the HTTP response sent by the controller.
     attr_internal :response_body
+
+    ##
+    # Returns the name of the action this controller is processing.
     attr_internal :action_name
+
+    ##
+    # Returns the formats processed by the controller.
```

Comment:
Yeah, I think you're right I'll update the PR. I was doing some testing and using the code @alexcameron89 suggested above to see when this accessor gets called and understand how it's used. At least when I try calling it from a controller, it doesn't seem to hit this accessor in AbstractController::Base. I think it hits the accessor in ActionView instead. When is this accessor for formats used?